packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,450 @@
+const StyledContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 40px 0 0 0;
+
+    #nft-transfer-modal {
+        width: 400px;
+    }
+
+    img {
+        width: 100% !important;
+        max-width: 300px;
+        border-radius: 8px;
+        margin-bottom: 16px;
+    }
+
+    .transfer-txt {
+        margin-top: 16px;
+    }
+
+    .confirm-txt {
+        margin-bottom: 4px !important; 
+    }
+
+    .confirm-img {
+        width: 172px;
+    }
+
+    .confirm-nft-card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: relative;
+
+        width: 100%;
+        margin-top: 16px;
+
+        background: #FFFFFF;
+        box-shadow: 0px 45px 56px rgba(0, 0, 0, 0.07), 0px 10.0513px 12.5083px rgba(0, 0, 0, 0.0417275), 0px 2.99255px 3.72406px rgba(0, 0, 0, 0.0282725);
+        border-radius: 8px;
+
+        .from-box {
+            position: relative;
+            width: 100%;
+            height: 74px;
+            border-top: 1px solid #F0F0F1;
+        }
+
+        .to-box {
+            position: relative;
+            width: 100%;
+            height: 74px;
+            border-top: 1px solid #F0F0F1;
+        }
+
+    }
+
+    .confirm-txt {
+        font-family: Inter;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 150%;
+        /* identical to box height, or 21px */
+        
+        font-feature-settings: 'zero' on;
+        
+        /* gray/neutral/600 */
+        
+        color: #72727A;
+
+        margin-left: 16px !important;
+    }
+
+
+    .h-right {
+        position: absolute;
+        right: 16px;
+        text-align: right;
+    }
+
+    .estimate-fee-card {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+
+        width: 100%;
+        height: 78px;
+        margin-top: 16px;
+
+        border: 1px solid #F0F0F1;
+        box-sizing: border-box;
+        border-radius: 8px;
+    }
+
+    .account-id {
+        font-family: Inter;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 14px;
+        line-height: 150%;
+        /* identical to box height, or 21px */
+        
+        text-align: right;
+        font-feature-settings: 'zero' on;
+        
+        /* gray/neutral/800 */
+        
+        color: #272729;
+    }
+
+    h3 {
+        font-family: Inter;
+        font-style: normal;
+        font-weight: 900;
+        font-size: 20px;
+        line-height: 130%;
+        /* identical to box height, or 26px */
+
+        text-align: center;
+
+        color: #24272A;
+    }
+
+    .success {
+        > p {
+            font-family: Inter;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 20px;
+            line-height: 150%;
+            /* or 30px */
+            
+            text-align: center;
+            font-feature-settings: 'zero' on;
```

Comment:
It seems we should add `font-feature-settings: 'zero' on;` to `GlobalStyle` instead since we want this site-wide?

Reply:
Actually please omit `font-feature-settings: 'zero' on;` completely as it seems @corwinharrell removed this style feature from the `GlobalStyle` not too long ago: https://github.com/near/near-wallet/commit/62c86e25362dc68d43487dbf84939d80918d649b